src/qt/coincontroldialog.cpp

```diff
@@ -118,10 +118,6 @@ CoinControlDialog::CoinControlDialog(const PlatformStyle *_platformStyle, QWidge
     // (un)select all
     connect(ui->pushButtonSelectAll, SIGNAL(clicked()), this, SLOT(buttonSelectAllClicked()));
 
-    // change coin control first column label due Qt4 bug.
```

Comment:
Oh, this can be removed completely?
I interpreted "first" as "otherwise it's needed later", that's why I didn't touch it.